### PR TITLE
Lots of fixes for the admin panel

### PR DESCRIPTION
**Pull request by NanoSector**
Mostly consistency fixes and cleanups here and there.

### CODE REVIEW

**Comment by jdarwood007** [comment]
@Gwenwyfar and @Antes Can you review these changes please.

**Review by Gwenwyfar — APPROVED** [review]

**Comment by Gwenwyfar** [comment]
Could you also reindent it where you haven't?

File: Themes/default/ManageSearch.template.php [hunk 1]
```diff
@@ -351,45 +341,43 @@ function template_spider_edit()
...
+				</dd>
+				<dt>
+					<strong><label for="spider_ip">', $txt['spider_ip_info'], ':</label></strong><br>
+					<span class="smalltext">', $txt['spider_ip_info_desc'], '</span>
+				</dt>
+				<dd>
+					<textarea name="spider_ip" id="spider_ip" rows="4" cols="20">', $context['spider']['ip_info'], '</textarea>
+				</dd>
+			</dl>
+			<hr>
```
Comment (XinYenFon) [inline Themes/default/ManageSearch.template.php:375]: SMF is not adding hr before buttons anymore.

File: Themes/default/Themes.template.php [hunk 2]
```diff
@@ -476,34 +465,33 @@ function template_set_settings()
...
-					<li>
-						<a href="', $scripturl, '?action=admin;area=theme;th=', $context['theme_settings']['theme_id'], ';', $context['session_var'], '=', $context['session_id'], ';sa=edit;filename=index.template.php">', $txt['theme_edit_index'], '</a>
-					</li>
-					<li>
-						<a href="', $scripturl, '?action=admin;area=theme;th=', $context['theme_settings']['theme_id'], ';', $context['session_var'], '=', $context['session_id'], ';sa=edit;directory=css">', $txt['theme_edit_style'], '</a>
-					</li>
-				</ul>
-			</div>';
+				<div class="title_bar">
+					<h3 class="titlebg config_hd">
```
Comment (XinYenFon) [inline Themes/default/Themes.template.php:474]: config_hd is a big icon and its not really fitting at title_bar/titlebg, maybe its best to remove the icons.
cc: @Gwenwyfar

**Review by XinYenFon — CHANGES_REQUESTED** [review]

File: Themes/default/ManageSearch.template.php (hunk 1 above)
Comment (NanoSector) [reply Themes/default/ManageSearch.template.php:375]: That's not something I added, but I'll remove them in the next commits.

Comment (XinYenFon) [reply Themes/default/ManageSearch.template.php:375]: Yea I know :P that's probably something slipped while we are cleaning templates.

File: Themes/default/Themes.template.php (hunk 2 above)
Comment (Gwenwyfar) [reply Themes/default/Themes.template.php:474]: Not relevant to the PR IMO, just leave it be for now.

**Pull request merged**